problems/objects.js

```diff
@@ -168,7 +221,20 @@ const cubeObj = () => {};
  * @returns {Object} Counts of e and a. {a: 2, e: 1}
  */
 
-const countAandE = () => {};
+const countAandE = (str) => {
+  let stringObj = { a: 0, e: 0 };
+  let array = str.split("");
+  console.log(array);
+  for (let i = 0; i < array.length; i++) {
+    console.log(str[i]);
```

Comment:
Try to remove console logs after you've finished with them. It's like cleaning up after yourself.